shared/src/taxClearanceCertificateResponse.ts

```diff
@@ -0,0 +1,25 @@
+import { UserData } from "./userData";
```

Comment:
This file and `taxClearanceCertificate.ts` were previously combined, but importing `userData` was causing a circular dependency.